src/fable/Fable.Core/AST/AST.Fable.fs

```diff
@@ -104,12 +104,13 @@ and MemberKind =
     | Setter
     | Field
 
-and Member(name, kind, argTypes, returnType, ?genParams, ?decorators,
+and Member(name, kind, argTypes, returnType, ?fullType, ?genParams, ?decorators,
            ?isPublic, ?isMutable, ?isStatic, ?hasRestParams, ?overloadIndex) =
     member x.Name: string = name
     member x.Kind: MemberKind = kind
     member x.ArgumentTypes: Type list = argTypes
     member x.ReturnType: Type = returnType
+    member x.FullType: Type = defaultArg fullType (Function(argTypes, returnType))
```

Comment:
As commented above, `FullType` seems to indicate there're "partial types" in Fable. To avoid that but still convey the fact that this is not the actual type as parameters are not curried in the compiled JS, I'd suggest something like: `OriginalType`, `FSharpType` or `CurriedType` ...maybe the best option is the last one with a comment explaining that arguments are not actually curried in compiled code.